cura/CuraApplication.py

```diff
@@ -1999,8 +2000,11 @@ def _readMeshFinished(self, job):
                         center_y = 0
 
                     node.translate(Vector(0, center_y, 0))
-
                     nodes_to_arrange.append(node)
+            # If file extention is 3mf and models are to be loaded from a cura project,
```

Comment:
This isn't entirely true. Currently we only have project files that are 3mf, but that is not a given. 

It's more correct to say "If the file is a project, but you want to open as a model, the files should be arranged" (so basically removing any reference to 3MF here).

Reply:
fixed 30e4c61620fc31703cf0b4227f419beea882db30